cypress/integration/userCanLogin.spec.js

```diff
@@ -0,0 +1,14 @@
+describe('User can log in to application', () => {
+
+  it('successfully logs in ', () => {
+    cy.loginUser('user@mail.com', 'password')
+    cy.get('#welcome-message')
+      .should('contain', 'Hello BettySpaghetti')
+  })
+
+  it('unsuccessfully logs in ', () => {
+    cy.failToLoginUser('user@mail.com', 'passwordd')
+    cy.get('#response-message')
+      .should('contain', 'You need to sign in or sign up before continuing.')
```

Comment:
If you provide wrong credentials, the error message that the Rails API sends back is NOT `You need to sign in or sign up before continuing.`

Reply:
Yeah, I know. Thanks for noticing. I was reviewing all kinds of PRs and got mixed up in my head.